Cue/app/tabs/library/DeckView.js

```diff
@@ -12,26 +12,42 @@ import type { Card, Deck } from '../../api/types'
 import CueColors from '../../common/CueColors'
 import CueHeader from '../../common/CueHeader'
 import CueIcons from '../../common/CueIcons'
+import ListViewHairlineSeparator from '../../common/ListViewHairlineSeparator'
+import ToolbarIOS from '../../common/ToolbarIOS'
+
+import DeckViewInfoHeader from './DeckViewInfoHeader'
+import DeckViewRow from './DeckViewRow'
 
 const styles = {
   container: {
     flex: 1,
     backgroundColor: 'white'
   },
-  bodyContainer: {
-    flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center'
-  },
-  button: {
-    color: CueColors.primaryTint
-  },
 }
 
-export default class DeckView extends React.Component {
-  props: {
-    navigator: Navigator,
-    deck: Deck
+type Props = {
+  navigator: Navigator,
+  deck: Deck,
+  userId: string
```

Comment:
nevermind ignore these comments, i see how this works now. The connect from react-redux sets the userId prop.